src/visualizers/Grid.ts

```diff
@@ -397,8 +397,8 @@ const propertyIndicatorFunction: {
     [key in PropertyName]: key extends FactorPropertyName
         ? (factorization: Factorization) => boolean
         : key extends ValuePropertyName
-        ? (() => boolean) | ((value: bigint, aux?: bigint) => boolean)
-        : never
+          ? (() => boolean) | ((value: bigint, aux?: bigint) => boolean)
+          : never
```

Comment:
why spacing?

Reply:
Actually, this reflects a change in how our automated tools reformat Grid.ts that apparently resulted from the giant dependency update we have been going through. I am not quite sure how it filtered through to be a diff in this PR, but now that I have rebased, it is gone as a difference.